Kernel/Arch/aarch64/linker.ld

```diff
@@ -30,7 +30,6 @@ SECTIONS
         end_of_safemem_atomic_text = .;
 
         *(.text*)
-        end_of_kernel_text = .;
```

Comment:
Did you intentionally remove this here?

Reply:
I moved it to the end of `.unmap_after_init`. That matches the position of the symbol on x86-64 and riscv64.
(.unmap_after_init is also part of the text pheader)